src/collections/string.c

```diff
@@ -103,6 +103,8 @@ void _z_str_free(char **src) {
     }
 }
 
+void _z_str_copy(char *dst, const char *src) { strncpy(dst, src, strlen(src)); }
```

Comment:
So this is just `strcpy`? Is this an attempt at bypassing MISRA's forbidding of it? :p